native/membrane_videocompositor/src/compositor/texture_transformations/corners_rounding/mod.rs

```diff
@@ -0,0 +1,44 @@
+use bytemuck::{Pod, Zeroable};
+
+use crate::compositor::VideoProperties;
+
+use super::TextureTransformation;
+
+/// Struct representing parameters for video corners rounding texture transformation.
+/// corner_rounding_radius is [0, 1] range float representing the radius of the circle "cutting"
+/// frame corner part. [0, 1] range is mapped into pixels based on video width, meaning
+/// corner_rounding_radius equals 0.1 in FullHD video makes 192 pixels long radius of circles.
+#[derive(Debug, Clone, Copy, Zeroable, Pod, PartialEq)]
+#[repr(C)]
+pub struct CornersRounding {
+    pub corner_rounding_radius: f32,
+    pub video_width: f32,
+    pub video_height: f32,
+}
```

Comment:
This isn't implemented properly.....

Reply:
I implemented `border_radius` as a radius in pixels, accordingly to slack survey results.